.github/workflows/anaconda_amazonlinux_ci.yml

```diff
@@ -31,15 +35,17 @@ jobs:
         uses: docker/setup-qemu-action@29109295f81e9208d7d86ff1c6c12d2833863392 # v3.6.0
         with:
           # https://github.com/docker/setup-qemu-action/issues/188#issuecomment-2604322104
-          image: tonistiigi/binfmt:qemu-v8.1.5
+          image: tonistiigi/binfmt:qemu-v8.1.5@sha256:2d2918e86e5327d0661f7083d67a95280b0f7be8f77ed79a8418f81d7d90ce6f
           platforms: linux/arm64/v8
 
       - name: Set up Docker Buildx
         id: buildx
         uses: docker/setup-buildx-action@b5ca514318bd6ebac0fb2aedd5d36ec1b5c232a2 # v3.10.0
         with:
-          version: latest
+          cache-binary: false
           driver-opts: network=host
+          # renovate: datasource=github-releases depName=docker/buildx
+          version: v0.23.0
```

Comment:
seems like for complete pinning buildkit also has to be pinned, see https://docs.docker.com/build/ci/github-actions/configure-builder/#version-pinning